cadence/values.yaml

```diff
@@ -57,6 +57,85 @@ server:
+    # clusterMetadata contains the settings of the Cadence clusters
+    # participating in a Cadence cluster group.
+    clusterMetadata:
+      # enableGlobalDomain when set to true allows creating global domains which
+      # are replicated between all Cadence clusters distributed among different
+      # Kubernetes clusters and whose tasks are served by the active Cadence
+      # cluster in the replication.
+      #
+      # WARNING: This value **MUST NOT** be changed after launching the first
+      # Cadence cluster of the Cadence cluster group.
+      enableGlobalDomain: true
+
+      # maximumClusterCount defines and locks the maximum number of Cadence
+      # clusters of a Cadence cluster group at any point in the future included
+      # clusters added later.
+      #
+      # The requirement for this is to be able to safely determine a
+      # failoverVersionIncrement and initialFailoverVersion for each cluster
+      # automatically, accounting for clusters which are potentially being added
+      # later on to the cluster group and ensuring unique failover versions for
+      # each cluster all the time.
+      #
+      # failoverVersionIncrement is the increment of each cluster version when
+      # failover happens.
+      #
+      # initialFailoverVersion is the failover version a cluster starts at when
+      # launched.
+      #
+      # WARNING: For each Cadence cluster, (initialFailoverVersion +
+      # i*failoverVersionIncrement) **MUST** result in a unique value, not used
+      # by any other cluster.
+      #
+      # WARNING: This value **MUST NOT** be changed after launching the first
+      # Cadence cluster of the Cadence cluster group.
+      maximumClusterCount: 100
+
+      # masterClusterName is the name of the Cadence leader cluster in a Cadence
+      # cluster group which can register a domain, while each cluster can update
+      # domains and do domain failover.
+      #
+      # WARNING: it **MUST** be set to the same value for all Cadence clusters
+      # in a Cadence cluster group.
+      masterClusterName: "cluster-0"
+
+      # currentClusterName is the name of the Cadence cluster the configuration
+      # belongs to.
+      #
+      # In a Cadence cluster group, this is the particular Cadence
+      # cluster being installed with the chart on a Kubernetes cluster.
+      #
+      # WARNING: This value **MUST NOT** be changed after launching the Cadence
+      # cluster.
+      currentClusterName: "cluster-0"
+      
+      # clusterInformation contains the information about each Cadence cluster
+      # which participates in a Cadence cluster group.
+      #
+      # WARNING: for each Cadence cluster at any moment, the cluster names,
+      # initialFailoverVersion values and (initialFailoverVersion +
+      # i*failoverVersionIncrement) **MUST** be a unique value, not used by any
+      # other cluster.
+      #
+      # WARNING: you **MUST** list the Cadence clusters of a Cadence cluster
```

Comment:
Last comment: can we add a comment here that only adding item to the list is allowed? Removing would cause the failover version to mess up.

Reply:
Fixed in https://github.com/banzaicloud/banzai-charts/compare/d7a21e7928d6a552bb021bfc742df24012f6afe7..febd48f1e03e8fbd63924ffd4c7c3cdd1a0be69c and https://github.com/banzaicloud/banzai-charts/compare/febd48f1e03e8fbd63924ffd4c7c3cdd1a0be69c..5a0499cdcee74f4d8808cb4e9174b9da920d8301.